lib/ui/src/core/layout.js

```diff
@@ -149,6 +156,17 @@ export default function({ store }) {
       });
     },
 
+    focusOnUIElement(elementId) {
+      const focusableElementId = focusableUIElements[elementId];
+      if (focusableElementId == null) {
```

Comment:
nit: even if `undefined == null` can we change this check to be `!focusableElements`